studio/data/projects/project-readonly-status-query.ts

```diff
@@ -0,0 +1,55 @@
+import { useQuery, useQueryClient, UseQueryOptions } from '@tanstack/react-query'
```

Comment:
shouldn't this just come down on another query?

like if i was using this API, i'd expect to see `readOnly` in the response for something like `/projects` not have to do an individual query.